### PR TITLE
Add elixir SDK

### PR DESCRIPTION
**Pull request by giraffekey**
Hi! I created a basic SDK to support Elixir.

As you can see at the bottom of [this page](https://opentelemetry.io/docs/languages/erlang/instrumentation/), the logs and metrics APIs are still unstable, so I didn't use them.

Let me know if there's anything you'd like me to change or add.

/claim #5082 

Closes HIG-3880

### CODE REVIEW

**Comment by giraffekey** [comment]
Updates:

- Added `init/0` function
- Added log event tracing
- Expanded documentation
- Added a README.md

File: sdk/highlight-ex/lib/highlight.ex [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      if request_id do
+        Tracer.set_attributes([{:"highlight.trace_id", request_id}])
+      end
+
+      Tracer.record_exception(e, [])
+    end
+  end
+
+  defp base_attributes(config) do
+    [{:"highlight.project_id", config.project_id}] ++
```
Comment (Vadman97) [inline sdk/highlight-ex/lib/highlight.ex:85]: could we add the telemetry distro attributes to help identify this sdk? https://opentelemetry.io/docs/specs/semconv/resource/#telemetry-sdk

File: sdk/highlight-ex/mix.exs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+defmodule Highlight.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :highlight,
+      version: "0.1.0",
```
Comment (Vadman97) [inline sdk/highlight-ex/mix.exs:7]: how do we distribute this package? anything we need to add to CI to publish this SDK automatically?

File: sdk/highlight-ex/test/highlight_test.exs
Comment (Vadman97) [inline sdk/highlight-ex/test/highlight_test.exs:1]: could we run this in CI automatically?

File: sdk/highlight-ex/lib/highlight.ex [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+  @doc """
+  Initialize the Highlight SDK with the given configuration.
+  This sets up Elixir for automatic log collection.
+
+  ## Examples
+
+    iex> Highlight.init()
+  """
+  def init() do
```
Comment (Vadman97) [inline sdk/highlight-ex/lib/highlight.ex:25]: could we add docs on how to set up the SDK, ie. https://github.com/highlight/highlight/blob/64827f23e091bed85e79756e5370745a26337b12/highlight.io/components/QuickstartContent/traces/dotnet/dot-net.tsx#L30

**Review by Vadman97 — APPROVED** [review]
💪 a great starting point for the SDK! thank you for your contribution

**Comment by Nitin6404** [comment]
hey @giraffekey @Vadman97 i would like to help and improve this Elixir sdk, so do you have any further plans for it?

**Comment by Vadman97** [comment]
@giraffekey should be good to merge! could you please rebase on main so we can merge this?

**Comment by giraffekey** [comment]
I wasn't really sure how to rebase things so I might've messed things up

**Comment by Vadman97** [comment]
@giraffekey were there any commit here that were useful? i can isolate them for you if that would help

**Comment by giraffekey** [comment]
I'm going to create a new branch and add the files

**Pull request closed without merging**